java/opendcs/src/main/java/decodes/tsdb/ComputationApp.java

```diff
@@ -336,9 +336,9 @@ public void runApp( )
 			while(!shutdownFlag)
 			{
 				Logger.instance().debug3("ComputationApp start of main loop.");
-				try(
-					TimeSeriesDAI timeSeriesDAO = theDb.makeTimeSeriesDAO();
-					LoadingAppDAI loadingAppDAO = theDb.makeLoadingAppDAO();
+				try(// TODO: This is a good place to hook a transaction. Still need to be implemented though before we can demo it
```

Comment:
@ktarbet here is some of the initial usage you were interested from the "access-key" PR.

Implementation is in `SimpleOpenDcsDatabaseWrapper`.

Still need to create the transaction, but I think that might come in a separate PR. We've gotten good feedback on the general design and the idea was always to merge into main as soon a practical so all of the finalizing changes are in smaller PRs.